packages/react-composites/src/composites/CallWithChatComposite/adapter/AzureCommunicationCallWithChatAdapter.ts

```diff
@@ -902,7 +941,15 @@ export class TeamsMeetingLinkProvider implements ChatThreadProvider {
     this.callAdapterPromise = callAdapterPromise;
   }
 
-  public async getChatThread(): Promise<string> {
+  public isCallInfoRequired(): boolean {
+    return true;
+  }
+
+  public getChatThread(): string {
+    return getChatThreadFromTeamsLink(this.locator.meetingLink);
```

Comment:
This should only now happen if not `/** @conditional-compile-remove(meeting-id) */`. We want to _always_ get the thread Id from the call object when joining via Teams interop.

```suggestion
    /** @conditional-compile-remove(meeting-id) */
    throw new Error('Chat thread ID is not available for Teams meeting ID');
    return getChatThreadFromTeamsLink(this.locator.meetingLink);
```